vscode-cedar-wasm/src/policy.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use cedar_policy::{Policy};
+use serde::{Deserialize, Serialize};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen(typescript_custom_section)]
+const EXPORT_POLICY_RESULT: &'static str = r#"
+export class ExportPolicyResult {
+  free(): void;
+  readonly success: boolean;
+  readonly json: string | undefined;
+}"#;
+
+#[wasm_bindgen(getter_with_clone, skip_typescript)]
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ExportPolicyResult {
+    #[wasm_bindgen(readonly)]
+    pub success: bool,
+    pub json: Option<String>,
+}
+
+#[wasm_bindgen(js_name = exportPolicy)]
+pub fn export_policy(input_policy_str: &str) -> ExportPolicyResult {
+  let policy = match Policy::parse(None, input_policy_str) {
+    Ok(policy) => policy,
+    Err(_err) => {
+      return ExportPolicyResult {
+        success: false,
+        json: None,
+      }
+    }
+  };
+
+  let json = policy.to_json().unwrap();
```

Comment:
instead of `.unwrap()`, this should return an error (`ExportPolicyResult` with `success: false`) to avoid crashing things.

```suggestion
  let Ok(json) = policy.to_json() else {
      return ExportPolicyResult {
        success: false,
        json: None,
      };
  };
```